composable/assetlist.json

```diff
@@ -21,7 +21,24 @@
         "logo_URIs": {
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/composable/images/composable.svg"
         },
-        "coingecko_id": "composable-finance-layr"
+        "coingecko_id": "picasso-network"
```

Comment:
this is not a valid coingecko ID https://api.coingecko.com/api/v3/coins/list
Please remove

Reply:
removed thanks